osu.Game/Online/Spectator/SpectatorScoreProcessor.cs

```diff
@@ -152,12 +152,12 @@ public void UpdateScore()
 
             scoreInfo.MaxCombo = frame.Header.MaxCombo;
             scoreInfo.Statistics = frame.Header.Statistics;
+            scoreInfo.MaximumStatistics = spectatorState.MaximumStatistics;
 
             Accuracy.Value = frame.Header.Accuracy;
             Combo.Value = frame.Header.Combo;
 
-            scoreProcessor.ExtractScoringValues(frame.Header, out var currentScoringValues, out _);
```

Comment:
This was the only usage of the `ExtractScoringValues(FrameHeader, out ScoringValues, out ScoringValues)` overload. Probably could get rid of it now?